lib/monitoring/middleware/conjur_exporter.rb

```diff
@@ -0,0 +1,94 @@
+require 'prometheus/client'
+require 'prometheus/client/formats/text'
+
+module Monitoring
+  module Middleware
+    # Exporter is a Rack middleware that provides a sample implementation of a
+    # Prometheus HTTP exposition endpoint.
+    #
+    # By default it will export the state of the global registry and expose it
+    # under `/metrics`. Use the `:registry` and `:path` options to change the
+    # defaults.
+    class ConjurExporter
+      attr_reader :app, :path, :registry
+
+      FORMATS  = [::Prometheus::Client::Formats::Text].freeze
+      FALLBACK = ::Prometheus::Client::Formats::Text
+
+      def initialize(app, options = {})
+        @app = app
+        @path = options[:path] || '/metrics'
+        @port = options[:port]
+        @registry = options[:registry] || Monitoring::Prometheus.registry
```

Comment:
My thoughts are that it might be better for us to explicitly pass in these values via the constructor and not have defaults. For 2 reasons
1. Avoid the reader having to look at both the constructor and the call site to understand the options used. For the explicit case they just need to look at the call site
2. This might be a weaker reason, but... Avoid implicitly coupling default option values with the exporter definition. 

I realise this is subjective so it's your choice how to proceed.